api/test/build/BUILD

```diff
@@ -5,6 +5,7 @@ licenses(["notice"])  # Apache 2
 api_cc_test(
     name = "build_test",
     srcs = ["build_test.cc"],
+    linkstatic = True,
```

Comment:
im wondering if we should add a comment with the upstream issue that requires this